packages/zoe/src/zoeService/zoeStorageManager.js

```diff
@@ -155,6 +154,195 @@ export const makeZoeStorageManager = (
+          );
+          state.instanceState.addIssuer(keyword, issuerRecord);
+          return issuerRecord;
+        },
+        makeZoeMint(
+          keyword,
+          assetKind = AssetKind.NAT,
+          displayInfo,
+          { elementShape = undefined } = {},
+        ) {
+          const { state, facets } = this;
+          // Local indicates one that zoe itself makes from vetted code,
+          // and so can be assumed correct and fresh by zoe.
+          const issuerBaggage = makeScalarBigMapStore('IssuerBaggage', {
+            durable: true,
+          });
+          const localIssuerKit = makeDurableIssuerKit(
+            issuerBaggage,
+            keyword,
+            assetKind,
+            displayInfo,
+            state.adminNode.terminateWithFailure,
+            { elementShape },
+          );
+          zoeMintBaggageSet.add(issuerBaggage);
+          return facets.helpers.wrapIssuerKitWithZoeMint(
+            keyword,
+            localIssuerKit,
+            state.adminNode,
+          );
+        },
+        registerFeeMint(keyword, allegedFeeMintAccess) {
+          const { state, facets } = this;
+          const feeIssuerKit = feeMint.getFeeIssuerKit(allegedFeeMintAccess);
+          return facets.helpers.wrapIssuerKitWithZoeMint(
+            keyword,
+            feeIssuerKit,
+            state.adminNode,
+          );
+        },
+        getInstanceRecord() {
+          const { state } = this;
+          return state.instanceState.getInstanceRecord();
+        },
+        getIssuerRecords() {
+          const { state } = this;
+          return issuerStorage.getIssuerRecords(
+            // the issuerStorage is a weakStore, so we cannot iterate over
+            // it directly. Additionally, we only want to export the
+            // issuers used in this contract instance specifically, not
+            // all issuers.
+            Object.values(
+              state.instanceState.getInstanceRecord().terms.issuers,
+            ),
+          );
+        },
+        initInstanceAdmin(instanceHandle, instanceAdmin) {
+          return instanceAdminStorage.updater.initInstanceAdmin(
+            instanceHandle,
+            instanceAdmin,
+          );
+        },
+        deleteInstanceAdmin(i) {
+          instanceAdminStorage.updater.deleteInstanceAdmin(i);
+        },
+        makeInvitation(
+          handle,
+          desc,
+          customProps = undefined,
+          proposalShape = undefined,
+        ) {
+          const { state } = this;
+
+          // If the contract-provided customProperties include the
+          // properties 'description', 'handle', 'instance',
+          // 'installation', their corresponding
+          // values will be overwritten with the actual values. For
+          // example, the value for `instance` will always be the actual
+          // instance for the contract, even if customProperties includes
+          // a property called `instance`.
```

Comment:
```suggestion
          // If the contract-provided `customProps` include the
          // properties 'description', 'handle', 'instance',
          // 'installation', their corresponding
          // values will be overwritten with the actual values. For
          // example, the value for `instance` will always be the actual
          // instance for the contract, even if `customProps` includes
          // a property called `instance`.
```
Also, the people who need this information probably won't have read this comment, and might be surprised. Should we surface the issue by e.g. rejecting `customProps` attempting to include the reserved fields?
```js
        makeInvitation(
          handle,
          desc,
          customProps = {},
          proposalShape = undefined,
        ) {
          const { state } = this;
          const standardProps = {
            description: desc,
            handle,
            instance: state.instanceState.getInstanceRecord().instance,
            installation: state.instanceState.getInstanceRecord().installation,
          };
          const collidingKeys = Reflect.ownKeys(customProps).filter(key => {
            return Reflect.has(standardProps, key);
          });
          collidingKeys.length === 0 ||
            Fail`custom properties must not collide with standard properties: ${q(collidingKeys)}`;
          const mergedProps = { ...customProps, ...standardProps };
          const invitationAmount = AmountMath.make(invitationKit.brand, harden([mergedProps]));
          if (proposalShape !== undefined) {
            proposalShapes.init(handle, proposalShape);
          }
          return invitationKit.mint.mintPayment(invitationAmount);
        },
```
Or (even more forward-looking) reserve an entire prefix for system-generated properties to avoid collision?

Reply:
Good idea, but not for this PR.  Added issue #6782.